### PR TITLE
[CLA] METRUM signature of Corporate CLA 1.0

### PR DESCRIPTION
**Pull request by JonathanNEMRY**

### CODE REVIEW

File: doc/cla/corporate/metrum.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+declaration.
+
+Signed,
+
+Jonathan Nemry jonathan.nemry@metrum.lu https://github.com/JonathanNEMRY
+
+List of contributors:
+
+Pascal Pelzer pascal.pelzer@metrum.lu https://github.com/PascalPelzer
+Jonathan Nemry jonathan.nemry@acsone.eu https://github.com/JonathanNEMRY
```
Comment (lmignon) [inline doc/cla/corporate/metrum.md]: @JonathanNEMRY are you sure? `jonathan.nemry@acsone.eu` :smirk:

Comment (JonathanNEMRY) [reply doc/cla/corporate/metrum.md]: :+1:  haha an old habit! :smile: thanks for the catch!

**Comment by mart-e** [comment]
Hello,

Thank you for the signature, it has been integrated into Odoo at revision https://github.com/odoo/odoo/commit/97eb4a36b0a0b1004e8c49d5a457360ea3d4cdcb.
Future pull requests made on odoo will be validated by the runbot.
Regards

**Comment by JonathanNEMRY** [comment]
many thanks!

**Pull request merged**